source/quick-start-sinatra.txt

```diff
@@ -34,6 +34,12 @@ read and write operations on the data in your cluster.
 Ruby. By using {+odm+}, you can easily interact with your data and
 create flexible data models.
 
+Sinatra is a Domain Specific Language (DSL) for creating web
```

Comment:
I: [Doesn't look like this needs capitalization](https://en.wikipedia.org/wiki/Domain-specific_language)

```suggestion
Sinatra is a domain-specific language (DSL) for creating web
```